src/datachain/lib/dc.py

```diff
@@ -411,7 +484,7 @@ def jmespath_to_name(s: str):
             object_name = jmespath_to_name(jmespath)
         if not object_name:
             object_name = meta_type
-        chain = DataChain.from_storage(path=path, type=type, **kwargs)
+        chain = DataChain.from_storage(uri=path, type=type, **kwargs)
```

Comment:
Rename `path` param to `uri` as well?

Reply:
This is another method so I would like to leave this to another PR / refactoring as this one is already too big ... but I agree with you, it should be `uri` there as well